controller_interface/include/controller_interface/controller_interface.hpp

```diff
@@ -48,7 +48,7 @@ class ControllerInterface : public rclcpp_lifecycle::node_interfaces::LifecycleN
   virtual
   return_type
   init(
-    std::weak_ptr<hardware_interface::RobotHardware> robot_hardware,
+    std::weak_ptr<resource_manager::ResourceManager> resource_manager,
```

Comment:
is the resource manager in its own package? Or why the namespace "resource_manager::`?

Reply:
Namespace != package

Having a separate `resource_manager` namespace could help keeping it separate from the rest of the components. If we don't foresee having multiple things in it though than it's probably not worth.

Reply:
it is. The namespace all across ROS2 resembles the folder structure. it's package name plus subfolders.